packages/xo-server-test/src/backupNg/backupNg.spec.js

```diff
@@ -446,7 +446,7 @@ describe('backupNg', () => {
   })
 
   test('execute three times a delta backup with 2 remotes, 2 as retention and 2 as fullInterval', async () => {
-    jest.setTimeout(6e4)
+    jest.setTimeout(12e6)
```

Comment:
3 hours????

Reply:
My bad it's 12e5.

Reply:
Fixed ba54b5319467a40eaf11d164d51b76baf20f2d33